Update New-AzureRmDiskUpdateConfig call

jacobjiggler opened the pull request:
which fixes this error you get by following the current instructions.
Update-AzureRmDisk : Required parameter 'disk.Properties' is missing (null).

See comments here.
https://docs.microsoft.com/en-us/azure/virtual-machines/windows/convert-disk-storage?hubRefSrc=email&utm_source=lfemail&utm_medium=email&utm_campaign=lfnotification

azurecla commented:
Hi __@pelagon__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jacobjiggler committed:
Update New-AzureRmDiskUpdateConfig call

which fixes this error you get by following the current instructions.
Update-AzureRmDisk : Required parameter 'disk.Properties' is missing (null).
